crates/aptos-faucet/src/main.rs

```diff
@@ -271,30 +272,35 @@ mod tests {
         if let Script(script) = txn.payload() {
             panic!("unexpected type of script: {:?}", script.args())
         }
-        if let Some(script_function) = ScriptFunctionCall::decode(txn.payload()) {
-            match script_function {
-                ScriptFunctionCall::AccountCreateAccount {
-                    auth_key: address, ..
-                } => {
-                    let mut writer = accounts.write();
-                    let previous = writer.insert(address, AccountState::new(0));
-                    assert!(previous.is_none(), "should not create account twice");
+        if let ScriptFunction(script_function) = txn.payload() {
+            // We have to manually match mint function calls since the transaction builder
+            // cannot generate encoders/decoders generic Coin::mint<CoinType> functions.
```

Comment:
hmm, I thought the transaction builder can parse generics (ty_args)

Reply:
I built the transaction builder and it pretty much skipped the entire Coin module because all functions there are type generic unfortunately.

Reply:
No, it cannot, isn't that why you had to exclude genesis.move @zekun000 ?

Reply:
nope, that was because it can’t handle vector<vector<u8>> or vector<u64> etc